mathics/core/symbols.py

```diff
@@ -743,7 +745,12 @@ class BooleanType(SymbolConstant):
     the constant is either SymbolTrue or SymbolFalse.
     """
 
-    pass
+    @property
```

Comment:
I thiink @mmatera had mentioned that redfining True and False in MMA, if it is possible, will horribly break MMA.

I recall we had a discussion about whether the parent class SymbolConstant could be a literal. Right now it is not. However True and False I have a harder time seeing why it wouldn't be literal.